### DISCUSSION TITLE
Improve header form

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [esag-swiss/iDig-Webapp#19] [Ideas]
@AlZorba 
 Quand non connecté, on pourrait avoir le header comme actuellement. peut–être une petite icone 🚫 en plus

Et quand connecté, afficher un autre div, genre : `connecté à localhost en tant que idig ✅ `. Et avec un bouton pour se "deconnecter"

Le bouton "deconnecter" vide toutes les données chargées dans l'app (tranches etc)

Je propose aussi que quand pas connecté, l'interface n'affiche rien, pas les filtre ni la table, seulement un message "non connecté" par exemple

**Comment by AlZorba** [comment]
iDig server a introduit la gestion des permission par trenches. Il devrait maintenant être possible retrouver ces info avec la webapp

**Comment by AlZorba** [comment]
Fait il y a qq temps !
